### PR TITLE
Uploaded documents and pairings update

### PR DESCRIPTION
**Pull request by landovsky**
added POST uploaded_documents
added POST uploaded_documents/pairings
updated readme
configuration update (api_path => api_host, settable externally)

### CODE REVIEW

File: lib/pina.rb [hunk 1]
```diff
@@ -52,15 +56,25 @@ def set_defaults
...
     def base_url=(base_url)
       @base_url = base_url
     end
 
+    def schema
+      use_ssl ? 'https://' : 'http://'
+    end
+
+    def tenant?
+      @tenant.blank? ? '' : @tenant + '.'
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina.rb]: ok

File: lib/pina/collections/errors.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module Pina
+  module Collections
+    class Errors
+      include Virtus.model
+
+      attribute :status_code
+      attribute :message
+      attribute :errors, Array[::Pina::Models::Error]
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/collections/errors.rb]: ok

File: lib/pina/collections/errors.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+      def all
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/collections/errors.rb]: ok

File: lib/pina/models/nested_value.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+module Pina
+  module Models
+    module Coercions
+      class NestedValue < Virtus::Attribute
+        def coerce(value)
+          value.values.join(', ').to_s
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/models/nested_value.rb]: ok

File: lib/pina/models/petty_cash_disburstment.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
+require 'pina/models/petty_cash_disburstment_item'
+
+module Pina
+  module Models
+    class PettyCashDisburstment < Base
+
+      attribute :created_at
+      attribute :updated_at
+      attribute :creator, NestedValue
+      attribute :modifier
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/models/petty_cash_disburstment.rb]: ok

File: lib/pina/models/petty_cash_disburstment_item.rb [hunk 6]
```diff
@@ -0,0 +1,22 @@
+require 'pina/models/base'
+
+module Pina
+  module Models
+    class PettyCashDisburstmentItem
+      include Virtus.model
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/models/petty_cash_disburstment_item.rb]: ok

File: lib/pina/petty_cash_disburstment.rb [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+        response
+      end
+
+      def all(params = {})
+        response = Pina::RestAdapter.get(:petty_cash_disburstments, params)
+
+        unless response.ok?
+          return Pina::Collections::Errors.new(response.to_hash.merge(status_code: response.status_code))
+        end
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/petty_cash_disburstment.rb]: ok

File: lib/pina/rest_adapter.rb [hunk 8]
```diff
@@ -64,6 +95,14 @@ def url(resource, id_or_params)
       def prepare_params_for_request(params)
         params.map { |key, value| "#{URI::escape(key.to_s)}=#{URI::escape(value.to_s)}" }.join('&')
       end
+
+      def resource_with_namespace(resource)
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/rest_adapter.rb]: ok

File: lib/pina/uploaded_document.rb [hunk 9]
```diff
@@ -4,6 +4,15 @@
 module Pina
   class UploadedDocument
     class << self
+      def create(uploaded_document)
```
[the remark this answers is not in the document]
Comment (landovsky) [reply lib/pina/uploaded_document.rb]: ok

File: lib/pina/version.rb [hunk 10]
```diff
@@ -1,3 +1,3 @@
 module Pina
-  VERSION = '0.13.0'
+  VERSION = '0.13.1'
```
Comment (Masa331) [inline lib/pina/version.rb]: tady by se mela zvysit spis minor verze ne? Byly tam nejaky api zmeny ne?

**Pull request merged**